packages/react/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -25,8 +25,10 @@ const Breadcrumb = ({
   });
 
   return (
-    <nav className={className} {...rest}>
-      {children}
+    <nav aria-label="Breadcrumb">
+      <ol className={className} {...rest}>
```

Comment:
Do we want `className` here as opposed to on the nav?

Reply:
The class applies a flex (and nothing else) and since it needs to be on the parent of the children I left it there. I can apply a classname attribute if people need to target it for some reason. but the ol still needs to have the flex

Reply:
Great! Thanks for clarifying.